frontend/public/src/components/EnrolledItemCard_test.js

```diff
@@ -147,41 +147,39 @@ describe("EnrolledItemCard", () => {
     })
   })
 
-  it("doesn't renders the course passed label if course is not passed", async () => {
+  it("doesn't renders the course passed label if course is passed but no certificate", async () => {
     // If there are no grades at all
     let inner = await renderedCard()
     let coursePassed = inner
       .find(".enrolled-item")
       .find(".badge-enrolled-passed")
     assert.isFalse(coursePassed.exists())
 
-    // If the user has grades but not passed
+    // If the user has passed grades but no certificate
     const grade = makeLearnerRecordGrade()
-    grade.passed = false
+    grade.passed = true
     enrollmentCardProps.enrollment.grades = [grade]
     inner = await renderedCard()
     coursePassed = inner.find(".enrolled-item").find(".badge-enrolled-passed")
     assert.isFalse(coursePassed.exists())
   })
   ;[true, false].forEach(isSelfPaced => {
-    it("renders the course passed label based on self_paced and course passed", async () => {
+    it("renders the course passed label based on course certificate independent of course pacing", async () => {
```

Comment:
What is this specifically testing for?

Reply:
It checks that the "course passed" label is shown independently of whether the course is self paced or not. But I don't know the history of why that matters. Do you have any ideas?

Reply:
I don't think this should exist if the label is independent of whether the course is self-paced or not.